tdrs-backend/tdpservice/search_indexes/parsers/tanf_validators.py

```diff
@@ -0,0 +1,191 @@
+"""Validators for TANF data parser."""
+
+# T1 Category 2 TANF Fatal Edits
+# https://www.acf.hhs.gov/sites/default/files/documents/ofa/tanf_fatal_edits_sections_1_and_4.pdf
+
+def t1_003(model_obj):
+    """Validate model_obj.STRATUM."""
+    return _get_field_by_item_number(model_obj, '5') > 0 and _get_field_by_item_number(model_obj, '5') < 100
+
+def t1_006(model_obj):
+    """Validate report month."""
+    month = model_obj.RPT_MONTH_YEAR
+    return int(str(month)[0:4]) >= 1998
```

Comment:
I only used the _get_field_by_item_number a couple functions to show what it would look like. It will be easier to make the validator functions with _get_field_by_item_number, but they might be harder to read.

However, the field value could be put in the comment.

Reply:
Yeah, the readability is rough if we were to use that throughout.

I think this is a discussion point for tabletop on how to meld the model_obj created as well as the schema dictionary we now have so we can map the item number, name/description, and the value since the model won't have it. Perhaps we need a middleman class before we pass values off to the T1 model/document? Or can we bake in our "get" function into the model easily?

In an ideal world, how would we want to invoke this easily?

Psuedo-code thought:
```python
def t1_003(model obj):
  category=2
  item_number = '5'
  name, value = model_obj.get_field(item_number)
  if value not in conditionals:
    return { 'type': category, 'error_code': self.__name__, 'msg': msg, 'item_desc': name, 'item_id': item_number }
  else:
    return {}
```

Some of that metadata might be better handled at the layer above but yeah, this is a feeder into 1121.

Reply:
I like the idea of there being a field class object that holds a field value and an item number. That object will be stored on the data models 

```py
class ValuePair(models.Model):
   item_num = models.CharField(max_length=220)
   value = models.CharField(max_length=220)

class Translator(models.Model):
   field = models.ManyToManyField('ValuePair')
   ...
```

I have been thinking that we should make every field a char field instead of a number field. I believe there will be enough instances where the "numbers" are treated like char strings to warrant this. In cases where we really need an int, limlke when we do math or a comparison, we can utilize a to_int() or _repr_() function.